dominion/exceptions.py

```diff
@@ -12,9 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License
 
+class DoesNotExist(Exception):
```

Comment:
The exception class name will have more context if you rename it to `ContainerDoesNotExist`.

Reply:
I guess it's redundant. It's clear that Dominion runs Pieman containers. Otherwise we will have to rename `Failed` into `ContainerFailed` or `Interrupted` into `ContainerWasInterrupted`. It's ugly, isn't it?

Reply:
Okay, I agree.